recipes/blis/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "spacy-blis" %}
+{% set version = "0.2.4" %}
+{% set sha256sum = "2d4ca1508fd6229c7994fc17ba324083a5b83f66612c8ea62623a41a1768b030" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/b/blis/blis-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  skip: true  # [win and py27]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - clangdev  # [win or osx]
+  host:
+    - python
+    - pip
+    - python
+    - setuptools
+    - numpy 1.15.*
+    - wheel >=0.32,<0.33
+  run:
+    - python
+    - numpy >=1.15.0
+
+test:
+  imports:
+    - blis
+    - blis.py
+    - blis.cy
+
+about:
+  home: http://github.com/explosion/cython-blis
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Fast matrix-multiplication as a self-contained Python library – no system dependencies!'
+
+  doc_url: https://github.com/explosion/cython-blis
+  dev_url: https://github.com/explosion/cython-blis
+
+extra:
+  recipe-maintainers:
+    - honnibal
+    - ines
```

Comment:
@ines, are you OK with being a maintainer here?

Reply:
@scopatz Yes 👍